docs/source/en/quantization/gptq.md

```diff
@@ -24,10 +24,19 @@ Try GPTQ quantization with PEFT in this [notebook](https://colab.research.google
 
 The [AutoGPTQ](https://github.com/PanQiWei/AutoGPTQ) library implements the GPTQ algorithm, a post-training quantization technique where each row of the weight matrix is quantized independently to find a version of the weights that minimizes the error. These weights are quantized to int4, but they're restored to fp16 on the fly during inference. This can save your memory-usage by 4x because the int4 weights are dequantized in a fused kernel rather than a GPU's global memory, and you can also expect a speedup in inference because using a lower bitwidth takes less time to communicate.
 
+Now, we are going to replace [AutoGPTQ](https://github.com/PanQiWei/AutoGPTQ) with [GPTQModel](https://github.com/ModelCloud/GPTQModel), the auto_gptq will be deprecated in the future.
+
```

Comment:
As I said previously in another thread, we are not really sure to deprecate auto-gptq for now. So let's introduce GPTQModel as a maintained fork of auto-gptq with newer features. Feel free to list the features that GPTQModel have compared to auto-gptq.

Reply:
@Qubitium

Reply:
@SunMarc  You mean list the features of GPTQModel that is different from AutoGPTQ in the `gptq.md` readme? Sure. I will write up it up now. 

For reference, GPTQmodel started as a fork of autogptq but for all intent and purposes, almost every single logic top-down including kernels have been re-written or refractored for both clarify, performance, and past/future model support.

Reply:
Fixed